### ISSUE TITLE
UI module Setup Issue

### ISSUE THREAD

**Issue by goldyliang** [crnk-project/crnk-framework#684]
I was trying to enable the crnk-UI in my project (/browse/), but looks like the instruction in the document is out-dated.

In the document, it says use below code to set up the UI module:
   UIModule operationsModule = UIModule.create(new UIModuleConfig());

But did not mention where should I put this set up code.  I searched through the crnk-example, and could not find the use of UIModule at all. I think the above code is for general case.

Because I am using it in spring boot, so by looking at the comment of the class CrnkUIAutoConfiguration, I figured out that what I need to do is just to add crnk-ui as compile dependency, and it starts to work.

The document can be more specific I believe.

**Comment by remmeier** [comment]
yes, since crnk supports most frameworks out there, setup procedure can differ a bit from framework to framework. Some more automated (spring) than others. There is http://www.crnk.io/releases/stable/documentation/#_integration_with_spring_and_spring_boot. But input where to put more information is welcomed.

**Comment by goldyliang** [comment]
How about under 3.7. Integration with Spring and Spring Boot,  
"The following auto configurations are available. **Refer to the comments of each auto configuration class for detail how the setup is integrated with Spring.**"
